docs/faq.md

```diff
@@ -204,6 +336,99 @@ You can consult booking made through Flight Create Orders using the Flight Order
+
+There are two things to consider regarding payments for flight booking:
+
+1. The payment between you (the app owner) and your customers (for the services provided + the price of the flight ticket). You decide how to collect this payment. It is not included in the API. A third-party payment gateway, like Stripe for example, will be the easiest solution for this.
+
+2. The payment between you and the consolidator (to be able to pay the airline and issue the flight ticket). This will be done between you and your consolidator of choice and is to be agreed upon with the consolidator.
+
+### How can I cancel a flight?
+
+Cancellation is possible with the Flight Orders Management API as long as the booking has not been issued by the consolidator yet. If the booking has been issued, it will need to be canceled by the consolidator directly.
+
+### How to make the airline consolidator wait before issuing a ticket?
+
+There is no standard way or dedicated element in the API request to do that. You will need to agree on the process with your consolidator. It can be adding a remark or a specific ticketing agreement.
+
+
+## Hotel Search
+
+### What are guarantee, deposit and prepay?
+
+* **Guarantee:** The hotel will save credit card information during booking but will not make any charges to the account. In the case of a no-show or out-of-policy cancellation, the hotel may charge penalties to the card.
+* **Deposit:** At the time of booking or on a given deadline, the hotel will charge the guest a percentage of the total amount of the reservation. The remaining amount is paid by the traveler directly at the hotel.
+* **Prepay:** The traveler must pay the total amount of the reservation during booking.
+
+### What is the total price?
+
+The price total refers to the total price to be paid for the full stay. The variations, on the other hand, represent the average price per night. In the example you highlighted, a guest will need to pay €548.56 for the three nights, and the average price for the room is €182.85 per night (€548.55 overall). For more details, you can refer to our data model found in the OpenAPI specification of the Hotel Search API.
+
+### What is the latest possible date for check-in?
+
+The maximum date for the 'checkInDate' parameter is 359 days from today. Anything beyond this will return the error message 'MAXIMUM ADVANCE DAYS BOOKING EXCEEDED'.
+
+### There is a mismatch in the amenities returned
+
+The amenities shown as examples in the API reference documentation may not match some of the results returned. The list of amenities in our documentation is the ones that we are able to index with our API. The hotel content is fed by the hotel providers, and this is not standard, as each hotel is free to add whatever content they want to the amenities list.
+
+### How to search a hotel by location
+
+Regarding the input for a specific location in a hotel search, you have the following options:
```

Comment:
We can no longer search by IATA codes with Hotel Search v3. In order to search by location you will need to use the third endpoint of Hotel List (/reference-data/locations/hotels/by-geocode), which allows you to search using a latitude & longitude, and returns hotelIds from the specific search coordinates. You will then need to use this hotelId in the third endpoint of the Hotel Search API.